src/macros.jl

```diff
@@ -128,19 +128,28 @@ function assert_validmodel(m, macrocode)
     end
 end
 
-
-const valid_senses = [:(<=), :≤, :(>=), :≥, :(==)]
```

Comment:
This is the only thing that broke JuMPeR, which is nothing in the scheme of things! I do have screens of ambiguities to fix though...